charts/tempo-distributed/README.md

```diff
@@ -250,13 +250,13 @@ Alternatively, in order to quickly test Tempo using the filestore, the [single b
 
 ```yaml
 config: |
-  auth_enabled: false
-    compactor:
-      compaction:
-        block_retention: 48h
-      ring:
-        kvstore:
-          store: memberlist
+  multitenancy_enabled: false
+  compactor:
+    compaction:
+      block_retention: 48h
+    ring:
+      kvstore:
+        store: memberlist
```

Comment:
```suggestion
    multitenancy_enabled: false
    compactor:
      compaction:
        block_retention: 48h
      ring:
        kvstore:
          store: memberlist
```
Actually, the rest of the blocks have one more indentation. I think we need to add it to `multitenancy_enabled` or remove it from everywhere else as with the compactor block.